App/screens/Home.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect, useContext } from 'react'
+import { FlatList } from 'react-native'
+import { CredentialEventType, CredentialState } from 'aries-framework'
+
+import AgentContext from '../contexts/AgentProvider'
+
+import { Button, SafeAreaScrollView, AppHeaderLarge, ModularView, NotificationListItem } from 'components'
+
+interface Props {
+  navigation: any
+}
+
+const Home: React.FC<Props> = ({ navigation }) => {
+  const agentContext = useContext<any>(AgentContext)
+
+  const [notifications, setNotifications] = useState<any>([])
+
+  const getNotifications = async () => {
+    const credentials = await agentContext.agent.credentials.getAll()
+    const credentialsToDisplay: any = []
+
+    credentials.forEach((c: any) => {
+      if (c.state !== CredentialState.Done) {
```

Comment:
Since I think we only want to display a notification for a credential that is in a state Offered (at least for now)? Or am I missing something here?
```suggestion
      if (c.state === CredentialState.OfferReceived) {
```